cocos/tween/actions/action-interval.ts

```diff
@@ -360,6 +376,19 @@ export class Sequence extends ActionInterval {
         }
         return null;
     }
+
+    isUnknownTime (): boolean {
+        if (this._actions.length === 0) return false;
+
+        const one = this._actions[0];
+        const two = this._actions[1];
+
+        if (this._last < 1) {
```

Comment:
Can change to `this_last === 0`

Reply:
We can't. Because there is another state, this._last == -1, which means all actions in Sequence were not updated.
So `< 1` here means `0` or `-1`.